modules/client-configuration/pages/bootstrap-repository.adoc

```diff
@@ -1,7 +1,7 @@
 [[client-cfg-tools-repository]]
 = Bootstrap Repository
 
-A bootstrap repository contains required packages for registering Salt or traditional clients during bootstrapping, as well as packages for installing Salt on clients.
+A bootstrap repository contains required packages for registering clients during bootstrapping, as well as packages for installing Salt on clients.
```

Comment:
Registering clients == installing + configuring Salt Minion (provided by Salt Bundle or the "classic" Salt package. For 5.0 we might restrict bootstrapping to Salt Bundle). For me, this sentence sounds a bit like a tautology.

Reply:
Yes, to avoid this, I now shortened the intro.